### PR TITLE
Update package_info_plus to 3.0.1

### PR DESCRIPTION
**Pull request by otopba**

### CODE REVIEW

**Comment by mateusfccp** [comment]
Hello, @otopba.

I am failing to understand what this PR does. It includes a lot of formatting changes, which is making hard to know what has indeed changed or not.

Could you clean this PR so that it respects the current code formatting configuration?

Also, could you provide more information regarding what does this PR proposes? It seems that it's related to Flutter 3.0, but this plugin should work fine with Flutter 3.0. Did you find any issue using it with Flutter 3.0?

**Comment by otopba** [comment]
Hello!
In this PR I updated all the dependencies and cleaned up the project =)
Applied code style

**Comment by mateusfccp** [comment]
I don't think the code style changes are necessary. I will kindly ask you to revert them. IMO, the current formatting is better, and it did not cause any issues with `pana` nor anything.

File: update_available/pubspec.yaml [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
 description: Know if there's any update for your Flutter app, based on published versions.
-version: 2.2.0-dev.1
+version: 3.0.0
 repository: https://github.com/mateusfccp/update_available/tree/master/update_available
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
-  flutter: ">=2.0.0"
+  sdk: ">=2.18.0 <3.0.0"
+  flutter: ">=3.0.0"
```
Comment (mateusfccp) [inline update_available/pubspec.yaml:8]: Do we have to break? I don't see any reason to require Flutter 3.0 if it already works with Flutter 3.0 and is also backwards compatible.

**Review by mateusfccp — CHANGES_REQUESTED** [review]

**Comment by otopba** [comment]
@mateusfccp 

Hi!
I really went a little overboard with the changes. Actually, all I had to do was upgrade package_info_plus to 3.0.1

**Comment by mateusfccp** [comment]
I am going to give some attention to the issues and PRs when my vacations start, which is 15/Dec.

**Comment by mateusfccp** [comment]
@otopba Could you rebase to the latest master? Now that #19 is merged we should not have any problem in the checks.

File: update_available/pubspec.yaml (hunk 1 above)
Comment (otopba) [reply update_available/pubspec.yaml]: Out of date

**Comment by otopba** [comment]
@mateusfccp fixed

**Review by mateusfccp — APPROVED** [review]

**Pull request merged**